internal/pkg/admission/policy-server-service.go

```diff
@@ -3,16 +3,37 @@ package admission
 import (
 	"context"
 	"fmt"
+	"os"
+	"strconv"
 
 	policiesv1alpha2 "github.com/kubewarden/kubewarden-controller/apis/policies/v1alpha2"
 
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 
 	"github.com/kubewarden/kubewarden-controller/internal/pkg/constants"
 )
 
+var (
+	// This is the port where the Policy Server service will be exposing metrics. Can be overriden
+	// by an environment variable KUBEWARDEN_POLICY_SERVER_SERVICES_METRICS_PORT
+	metricsPort = constants.PolicyServerMetricsPort
+)
+
+func init() {
+	envMetricsPort := os.Getenv(constants.PolicyServerMetricsPortEnvVar)
+	if envMetricsPort != "" {
+		var err error
+		metricsPort, err = strconv.Atoi(envMetricsPort)
```

Comment:
```suggestion
		metricsPort, err = strconv.ParseInt(envMetricsPort)
```

Reply:
Why so? `ParseInt` expects three arguments. `Atoi` is more handy, performs some safety checks and relies on `ParseInt` when necessary. Am I missing something?

Reply:
Ugh, I totally see the security issue here now. Thanks. Addressing it.